tools/cdn-purge/fastly-purge.sh

```diff
@@ -2,4 +2,4 @@
 
 cd tools/cdn-purge
 go mod vendor
-FASTLY_KEY=${FASTLY_KEY} go run fastly-purge.go -v
+FASTLY_KEY=${FASTLY_KEY} go run fastly-purge.go -v -b nr-downloads-main
```

Comment:
does it make sense to use the env variable `AWS_S3_BUCKET_NAME` ?

Reply:
I don't see any point as fastly is only for prod